src/Compilers/Core/Portable/Syntax/SyntaxList`1.cs

```diff
@@ -374,6 +374,16 @@ public bool Any()
             return _node != null;
         }
 
+        internal bool All(Func<TNode, bool> predicate)
+        {
+            foreach (var item in this)
+            {
+                if (!predicate(item)) return false;
```

Comment:
```suggestion
                if (!predicate(item))
                    return false;
```

Reply:
If ```return``` is moved on a separate line, ```{}``` should be used around it.

---
In reply to: [534342122](https://github.com/dotnet/roslyn/pull/49495#discussion_r534342122) [](ancestors = 534342122)